### ISSUE TITLE
custom driver to record mixed reality video

### ISSUE THREAD

**Issue by shensheng27** [ValveSoftware/openvr#633]

**Comment by maximeLong** [comment]
@shensheng27 did you ever make any progress on this question?

**Comment by shensheng27** [comment]
@maximeLong no.  Tried build demo app from scratch and using UE4 layers perform mixed reality video and it's working on that particular app, six months ago.
I think app developer is responsible for create mixed reality video using openvr. So there is little driver could do unless openvr  open such interface to upload camera image and render 2 view for player and viewer.
good luck
